include/preprocess/max_inscribed_ellipsoid_rounding.hpp

```diff
@@ -22,7 +22,8 @@ template
     typename Point   
 >
 std::tuple<MT, VT, NT> max_inscribed_ellipsoid_rounding(Polytope &P, 
-                                                        Point const& InnerPoint)
+                                                        Point const& InnerPoint,
+                                                        const bool aggressive_mode = false)
 {
     std::pair<std::pair<MT, VT>, bool> iter_res;
```

Comment:
We can add a todo here to change this to std::tuple.

Reply:
Thanks, I added a comment in `max_inscribed_ellipsoid.hpp`